### PR TITLE
`cookies_serializer` docs improvements

### PR DESCRIPTION
**Pull request by ghiculescu**
This PR incorporates @p8's feedback on https://github.com/rails/rails/pull/45137 and @rafaelfranca's on https://github.com/rails/rails/pull/45127#issuecomment-1136402456

### CODE REVIEW

File: railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt [hunk 1]
```diff
@@ -83,13 +83,16 @@
...
 
 # If you're upgrading and haven't set `cookies_serializer` previously, your cookie serializer
-# was `:marshal`. The default for new apps is `:json`.
+# is `:marshal`. The default for new apps is `:json`.
 #
-# You can convert all cookies to JSON using the `:hybrid` formatter. It is fine to use
-#`:hybrid` long term; you should do that unless you're confident that *all* your cookies
-# have been converted to JSON.
+# To migrate an existing application to the `:json` serializer set this option to `:hybrid`.
+# Rails transparently deserializes existing (Marshal-serialized) cookies on read and
```
Comment (rafaelfranca) [inline railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt:89]:
```suggestion
#
# Rails transparently deserializes existing (Marshal-serialized) cookies on read and
```

File: railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt [hunk 2]
```diff
@@ -83,13 +83,16 @@
...
+# re-writes them in the JSON format.
+# It is fine to use `:hybrid` long term; you should do that until you're confident *all* your cookies
```
Comment (rafaelfranca) [inline railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt:91]:
```suggestion
#
# It is fine to use `:hybrid` long term; you should do that until you're confident *all* your cookies
```

File: railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt [hunk 3]
```diff
@@ -83,13 +83,16 @@
 # Rails.application.config.active_storage.variant_processor = :vips
 
 # If you're upgrading and haven't set `cookies_serializer` previously, your cookie serializer
-# was `:marshal`. The default for new apps is `:json`.
+# is `:marshal`. The default for new apps is `:json`.
 #
```
Comment (rafaelfranca) [inline railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt:87]:
```suggestion
#
# Rails.application.config.action_dispatch.cookies_serializer = :json
```

**Review by rafaelfranca — COMMENTED** [review]
I made some suggestions on GItHub but they might be confusing if applied as-is.

The idea is to have 2 possible values for this config commented. First, the `:hybrid` value. Then, an explanation that `:json` is the default if you delete this file without configuring anything somewhere else. We could even use two different comment blocks.

**Comment by ghiculescu** [comment]
@rafaelfranca good idea. I applied those suggestions, and also moved it to the bottom of the file. I think this will make it easier to read and realise there's two options and you should consider both.

**Pull request merged**